File: C,C++/p12578.c
#include <stdio.h>
#include <math.h>

double luasLingkaran (double r);

int main () {
	int kasus, i;
	scanf("%d", &kasus);
	for (i = 0; i < kasus; i++) {
		int length;
		scanf("%d", &length);
		double luasLing = luasLingkaran((length*1.0)/5.0);
		double luasPersegi = ((6.0/10.0)*(length*1.0)) * (length*1.0);
		printf("%.2f %.2f\n", luasLing, (luasPersegi-luasLing));
	}
	return 0;
}

double luasLingkaran (double r) {
	return acos(-1)*r*r;
}

File: Java/p12578.java

import java.util.Scanner;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author DarKnight98
 */
public class p12578 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int kasus = input.nextInt();
        for (int i = 0; i < kasus; i++) {
            int length = input.nextInt();
            double luasLingkaran = luasLingkaran((length*1.0)/5.0);
            double luasPersegi = ((6.0/10.0)*(length*1.0)) * (length*1.0);
            System.out.printf("%.2f %.2f\n", luasLingkaran, (luasPersegi-luasLingkaran));
        }
    }
    
    static double luasLingkaran (double r) {
        return Math.acos(-1)*r*r;
    }
}
